integration-tests/src/tests/tools/dependencies.rs

```diff
@@ -18,11 +18,11 @@ use std::process::Command;
 use std::str;
 
 const LIBS_THRESHOLDS: [(&str, usize); 9] = [
-    ("near-primitives", 127),
-    ("near-jsonrpc-primitives", 134),
-    ("near-chain-configs", 130),
-    ("near-chain-primitives", 130),
-    ("near-client-primitives", 152),
+    ("near-primitives", 258),
```

Comment:
Extracting the code to `near-primitives` increased the number of dependencies, and the dependent libraries got larger too. I'm working on fixing this.